ref_builder/cli.py

```diff
@@ -108,28 +113,57 @@ def create(
         update_otu(repo, new_otu, ignore_cache=ignore_cache)
 
 
-@otu.command()
+@otu.command(name="update")
 @click.argument("TAXID", type=int)
 @debug_option
 @ignore_cache_option
 @path_option
-def update(debug: bool, ignore_cache: bool, path: Path, taxid: int):
+def otu_update(debug: bool, ignore_cache: bool, path: Path, taxid: int) -> None:
+    """Update an existing OTU with the latest data from NCBI."""
     configure_logger(debug)
 
     repo = Repo(path)
 
-    otu = repo.get_otu_by_taxid(taxid)
-    if otu is None:
+    otu_ = repo.get_otu_by_taxid(taxid)
+
+    if otu_ is None:
         click.echo(f"OTU not found for Taxonomy ID {taxid}.", err=True)
         click.echo(f'Run "virtool otu create {taxid} --autofill" instead.')
         sys.exit(1)
 
-    update_otu(repo, otu, ignore_cache=ignore_cache)
+    update_otu(repo, otu_, ignore_cache=ignore_cache)
+
+
+@otu.command(name="get")
+@click.argument("IDENTIFIER", type=int)
+@path_option
+def otu_get(identifier: str, path: Path) -> int:
+    """Get an OTU by its Taxonomy ID."""
+    try:
```

Comment:
Try-catch invalidated by `@click.argument("IDENTIFIER", type=int)`. Should we change the argument or the try-catch?